Documentation/contributing/release/feature.rst

```diff
@@ -64,14 +64,19 @@ On Freeze date
    #. ``needs-backport/1.2``
 
 
-#. Checkout to master and update the ``.github/cilium-actions.yml`` to have
-   all the necessary configurations for the backport of the new ``vX.Y`` branch.
+#. Checkout to master
+
+#. Bump the ``VERSION`` file to ``vX.Y+1.90`` and update the
+   ``.github/cilium-actions.yml`` to have all the necessary configurations for
+   the backport of the new ``vX.Y`` branch.
 
    ::
 
        git checkout -b pr/master-cilium-actions-update origin/master
        # modify .github/cilium-actions.yml
        git add .github/cilium-actions.yml
+       # modify VERSION
+       git add VERSION
```

Comment:
Also `make -C install/kubernetes` to propagate the new version throughout helm

Reply:
it turns out this is not needed it, it's done at a later stage, will revert